content/20.treatments.md

```diff
@@ -86,6 +86,68 @@ A list of current studies and their results, using carefully the information req
 
 Summarize the state of the antiviral approach.
 
+##### Hydroxychloroquine and Chloroquine
+
+Hydroxychloroquine (HCQ) and chloroquine (CQ) are quinolones that have been widely used as anti-malarial and autoimmune drugs in humans.
+HCQ and CQ act through the same mechanism of action, but HCQ has a better safety profile [@doi:10.1136/ard.2008.101766].
+HCQ/CQ have reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10.1016/S1473-3099(06)70361-9; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].
+In vitro studies of CQ use for SARs-CoV infection suggest anti-viral activity is due to elevation of endosomal pH and terminal glycosylation of angiotensin-converting enzyme 2, the cellular entry receptor [@doi: 10.1186/1743-422X-2-69].
+In vitro studies of SARS-CoV-2 infection show both HCQ and CQ to be effective, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
+Thus, there is much excitement about HCQ as a potential therapeutic for SARS-CoV-2 and COVID-19.
+This section will discuss current efforts on assessing the potential role of HCQ in treating COVID-19.
+
+###### Current Evidence
+
+Gautret et al conducted a non-randomized, non-blinded, non-placebo clinical trial on 42 hospitalized patients comparing HCQ to standard care [@doi:10.1016/j.ijantimicag.2020.105949].
+This trial found patients who received HCQ showed higher rates of virological clearance by nasopharyngeal swab on Days 3-6 when compared to standard care.
+This study also treated six patients with both HCQ + azithromycin and found this combination therapy to be more effective than HCQ alone.
+This study showed design and analysis weaknesses that severely limit interpretability of results.
+These weaknesses include: lack of randomization, lack of blinding, lack of placebo, lack of Intention-To-Treat analysis, lack of correction for sequential multiple comparisons, trial arms entirely confounded by hospital, false negatives in outcome measurements, lack of trial pre-registration, and small sample size.
+Two of these issues are due to inappropriate data analysis and can therefore be corrected post-hoc by recalculating p-values (lack of Intention-To-Treat analysis and multiple comparisons.)
+However, all other issues are fundamental design flaws and can not be corrected for.
+Thus, conclusions cannot be generalized outside of the study.
+Nevertheless, this study does provide promising preliminary data on the use of HCQ in COVID-19 treatment and supports further investigation of HCQ in large, rigorous clinical trials.
+
+A randomized, non-blinded, non-placebo trial of 62 COVID-19 patients at the Renmin Hospital of Wuhan University studied whether HCQ decreased time to fever break or time to cough relief when compared to standard care [@doi:10.1101/2020.03.22.20040758].
```

Comment:
I believe the trial was blinded. Quoting the paper: "Neither the research performers nor the patients
were aware of the treatment assignments. " This is the last sentence in the section "Study design and participants"

Reply:
Forgot to link the pdf: https://www.medrxiv.org/content/10.1101/2020.03.22.20040758v2.full.pdf

Reply:
I remember reading that and changing my mental note to "non-blinded" because without placebo, blinding is impossible. I should have made a note of the claim.

Trial arms were: standard care, standard care + HCQ. So at least physicians must know what they are giving and have potential to bias results. (Perhaps they pay more attention to HCQ patients. Or they are less aggressive in treating HCQ patients to give the drug time to work.) HCQ is an oral medication as well, so it's likely patients knew and could have improved due to placebo effect. 

Do you have ideas on how to succinctly and accurately phrase this distinction?

Reply:
Could just call this "open-label"?

Reply:
Oh that's a really good point... Perhaps you could note the claim, then add on the explanation you just mentioned here. I think that's a really useful bit of commentary, since people are a little confused about this trial. Derek Lowe's blog mentions it's blinded for example: https://blogs.sciencemag.org/pipeline/archives/2020/03/31/comparing-chloroquine-trials

Reply:
Done! Could you take a look?

Reply:
Just took a look. This expanded discussion looks great!